OpenRA.Game/Map/CellLayer.cs

```diff
@@ -45,9 +45,23 @@ public void CopyValuesFrom(CellLayer<T> anotherLayer)
 			Array.Copy(anotherLayer.entries, entries, entries.Length);
 		}
 
+		public static CellLayer<T> CreatePrototype(Func<int, int, T> creator, int mapWidth, int mapHeight, TileShape tileShape)
```

Comment:
@RoosterDragon from what you say I understand that I could join the mapheight and mapwidth variables into Sie to be consistent, but can't understand what you mean with the Func ambiguity stuff.